package/CHANGELOG

```diff
@@ -42,6 +42,7 @@ Enhancements
     (Issue #3994, PR #4281)  
   * Add support for reading chainID info from Autodock PDBQT files (Issue #4207,
     PR #4284)
+  * Add a new geometric selection: box (Issue #4323, PR #4324)
```

Comment:
We typically go new entries first, could you add this on line 33 instead?

Also please add yourself to the list of authors for 2.7.0 on line 18.